pre_commit_hooks/pretty_format_json.py

```diff
@@ -113,10 +120,22 @@ def main(argv=None):  # type: (Optional[Sequence[str]]) -> int
             )
 
             if contents != pretty_contents:
-                print('File {} is not pretty-formatted'.format(json_file))
+                print(
+                    'File {} is not pretty-formatted'.format(json_file),
+                    file=sys.stderr,
+                )
+                sys.stderr.flush()
 
                 if args.autofix:
                     _autofix(json_file, pretty_contents)
+                else:
+                    print(
+                        get_diff(
+                            ''.join(contents),
```

Comment:
this should just be `contents` right?

Reply:
🤔 I remember needing to join it before 

I guess not now that we're using `splitlines`?